themes/blue-owl.omp.json

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "final_space": false,
+  "osc99": true,
+  "console_title": true,
+  "console_title_style": "template",
+  "console_title_template": "{{if .Root}} ⚡ {{end}}{{.Folder | replace \"~\" \"🏚\" }} @ {{.Host}}",
+  "blocks": [
+    {
+      "type": "prompt",
+      "alignment": "left",
+      "segments": [
+        {
+          "type": "root",
+          "style": "plain",
+          "foreground": "#FFEB3B",
+          "background": "#a313a8",
+          "properties": {
+            "root_icon": "⚡"
+          }
+        },
+        {
+          "type": "os",
+          "style": "plain",
+          "foreground": "#ffffff",
+          "background": "transparent"
+        },
+        {
+          "type": "path",
+          "style": "diamond",
+          "leading_diamond": "<transparent,#01579B></>",
+          "trailing_diamond": "\uE0B0",
+          "foreground": "#ffffff",
+          "background": "#01579B",
+          "properties": {
+            "home_icon": "\uF7DB",
+            "folder_icon": "",
+            "folder_separator_icon": "<transparent> \ue0bd </>",
+            "style": "agnoster_short"
+          }
+        },
+        {
+          "type": "git",
+          "style": "powerline",
+          "powerline_symbol": "",
```

Comment:
Can you use the `\u` representation of icons? Make the search easier for people looking to replace/tweak. Thanks!

Reply:
Thanks, I have pushed an update with the \u representation of icons for Nerd Fonts icons.

Reply:
Thanks, I have pushed an update with the \u representation of icons for Nerd Fonts icons.